src/dictionaries/words

```diff
@@ -68,6 +68,7 @@ iterable
 jitter
 jobscript
 latin
+len
```

Comment:
In the normal course of events I'd prefer this to be a case-by-case exception directive, but I don't want tot try and insert such a thing into lint which also prints the CLI reference.

Reply:
Is `len` coming from a code example which hasn't been put in double backquotes? e.g:

```diff
- here's an example len(foo), this will
+ here's an example ``len(foo)``, this will
```

Reply:
I think that you are right. Cylc PR to follow then. 🙄

Reply:
```suggestion
```